smart_proxy_remote_execution_ssh.gemspec

```diff
@@ -0,0 +1,36 @@
+# -*- encoding: utf-8 -*-
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'smart_proxy_remote_execution_ssh/version'
+
+Gem::Specification.new do |gem|
+  gem.name          = "smart_proxy_remote_execution_ssh"
+  gem.version       = Proxy::RemoteExecution::Ssh::VERSION
+  gem.authors       = ['Ivan Nečas']
+  gem.email         = ['inecas@redhat.com']
+  gem.homepage      = "https://github.com/theforeman/smart_proxy_remote_execution_ssh"
+  gem.summary       = 'Ssh remote execution provider for Foreman Smart-Proxy'
+  gem.description   = <<-EOS
+    Ssh remote execution provider for Foreman Smart-Proxy
+  EOS
+
+  gem.files         = Dir['{bundler.d,lib,settings.d}/**/*', 'LICENSE', 'Gemfile']
+  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
+  gem.require_paths = ["lib"]
+  gem.license = 'GPLv3'
+
+  gem.add_development_dependency "bundler", "~> 1.7"
+  gem.add_development_dependency "rake", "~> 10.0"
```

Comment:
I end up getting this because the smart proxy already requires rake:

```
The gemspec at
/home/stbenjam/git/smart_proxy_remote_execution_ssh/smart_proxy_remote_execution_ssh.gemspec is not
valid. The validation error was 'duplicate dependency on rake (~> 10, development), (~> 10.0) use:
    add_runtime_dependency 'rake', '~> 10', '~> 10.0'
```